portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="content">
+        <h1>Hobbies</h1>
+        <nav>
+          <ul id="navigation">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="hobbies.html">Hobbies</a></li>
+          </ul>
+        </nav>
+        <div class="page-item">
+          <h2>Hiking</h2>
+          <img class="project-image" src="images/hike.jpg"/>
+          <p>
+            I love the outdoors and hiking! Last summer, I lived in the Adirondack Mountains in NY, and 
+            got to go camping, backpacking, and mountain climbing. This picture was taken at the peak of Mount Marcy,
+            the tallest peak in the state of New York! 
+          </p>
+        </div>
+        <div class="page-item">
+          <h2>Rock Climbing</h2>
+          <img class="project-image" src="images/climb.jpg"/>
+          <p>
+            Although I used to be afraid of heights, rock climbing is now one of my favorite hobbies. Cornell
+            has a rock wall on campus that I belong to, and love to top rope and boulder there!
+          </p>
+        </div>
+        <div class="page-item">
+          <h2>Running</h2>
+          <img class="project-image" src="images/run.jpg"/>
+          <p>
+            I have a love-hate relationship with running. Although sometimes my laziness gets the best of me, 
+            I love to run outdoors, especially when it is raining. This photo was taken at the Central Park Half Marathon 
+            in New York. I hope to run a full marathon one day! 
+          </p>
+        </div>
+        <div class="page-item">
+          <h2>Frisbee</h2>
+          <img class="project-image" src="images/frisbee.JPG"/>
+          <p> 
+            I am on the club frisbee team at Cornell! I love playing and tossing around the disk, but 
+            I mostly love the people I play with! Here is an action shot of me at a tournament in Buffalo, NY. 
+          </p>
+        </div>
+        <div class="page-item">
+          <h2>Scuba Diving</h2>
+          <img class="project-image" src="images/dive.jpg"/>
+          <p> 
+            This is a rather new hobby of mine, but I recently got PADI scuba dive certified! I love the water,
+            sea animals, and everything about scuba diving, and I can't wait to travel more and dive in new places.
+          </p>
+        </div>
+        <footer>
+          <ul id="footer">
+            <li><a href="https://linkedin.com/in/ali-kew">LinkedIn</a></li>
+            <li><a href="https://github.com/alikew24">Github</a></li>
+          </ul>
+        </footer>
+      </div>
```

Comment:
I can't remember the exact stlyistic pattern here, but I think a comment on the closing `<div>` is in order here. It reminds readers what large section of code is being closed. Something like `</div>  <!-- #content -->` (notice two spaces between the end of `<div>` and the start of `<!--`).

Basically, if I have to scroll a fair distance to try to guess what tag this one is closing, a comment is likely in order.